### PR TITLE
Integrate moin cluster into monitoring

### PR DESCRIPTION
**Pull request by michal-gubricky**

### CODE REVIEW

File: values-observer-scs.yaml [hunk 1]
```diff
@@ -28,13 +28,15 @@ thanos:
...
-## FIXME: Thanos query in Harbor cluster is temporary exposed via (unsecure) NodePort, see https://github.com/SovereignCloudStack/k8s-observability/issues/32
+## FIXME: Thanos query in Harbor and Moin cluster is temporary exposed via (unsecure) NodePort, see https://github.com/SovereignCloudStack/k8s-observability/issues/32
 ##   Thanos query should be exposed via ingress and then the store address below should be used.
 ##      - dnssrv+_http-harbor-cluster._tcp.thanos-query-envoy
 ## Uncomment this if you want to observe Harbor cluster via node port
       - 213.131.230.86:31050
-## FIXME: Thanos query in Harbor cluster is temporary exposed via (unsecure) NodePort, see https://github.com/SovereignCloudStack/k8s-observability/issues/32
-##   Thanos query in Harbor cluster should be exposed via ingress and then the section below should be used.
+## Uncomment this if you want to observe Moin cluster via node port
+      - 213.131.230.199:31050
```
Comment (chess-knight) [inline values-observer-scs.yaml:37]: If you want to be precise with the comments, I suggest adding a similar entry `dnssrv+`... for the moin cluster as a comment.
You also added the `thanosQueryEnvoySidecar.config` item on line 108. IMO also the `sidecars` and `extraVolumes` sections should be modified.

File: values-observer-scs.yaml
Comment (chess-knight) [inline values-observer-scs.yaml:1]: I think you are also missing `dnation-kubernetes-monitoring.clusterMonitoring.clusters` field

File: values-observer-scs.yaml (hunk 1 above)
Comment (michal-gubricky) [reply values-observer-scs.yaml:37]: Yes, miss that, thanks. But Why do you want to modify `extraVolumes`?

Comment (chess-knight) [reply values-observer-scs.yaml:37]: `extraVolumes` are probably not needed, sorry.

**Review by chess-knight — APPROVED** [review]

**Pull request merged**